interactions/api/models/user.py

```diff
@@ -65,6 +65,12 @@ def __init__(self, **kwargs):
     def __repr__(self) -> str:
         return self.username
 
+    def has_public_flag(self, flag: UserFlags) -> bool:
+        int(flag)
```

Comment:
What `69` line do?

Reply:
I don't remember why it's there but it's completely redundant. Removed it in my last commit.